testpar/t_filters_parallel.c

```diff
@@ -5554,16 +5379,23 @@ test_read_cmpd_filtered_dataset_type_conversion_unshared(void)
     hsize_t              block[READ_COMPOUND_FILTERED_CHUNKS_TYPE_CONVERSION_UNSHARED_DATASET_DIMS];
     hsize_t              flat_dims[1];
     size_t               i, read_buf_size, correct_buf_size;
-    hid_t                file_id = -1, dset_id = -1, plist_id = -1, filetype = -1, memtype = -1;
-    hid_t                filespace = -1, memspace = -1;
+    hid_t                file_id = H5I_INVALID_HID, dset_id = H5I_INVALID_HID, plist_id = H5I_INVALID_HID;
+    hid_t                filetype = H5I_INVALID_HID, memtype = H5I_INVALID_HID;
+    hid_t                group_id  = H5I_INVALID_HID;
+    hid_t                filespace = H5I_INVALID_HID, memspace = H5I_INVALID_HID;
```

Comment:
I usually put these one per line. I find them hard to parse when glommed together with commas.

Reply:
I mostly tried to just put related fields on the same line so that the function's declarations didn't get excessively long vertically, but I can always change this if it's hard to parse. With the introduction of H5S_BLOCK, I eliminated some old cruft variables, but didn't go over everything.